src/App.vue

```diff
@@ -21,15 +21,9 @@ export default {
     NavigationRegistered,
     Alert,
   },
-  data() {
```

Comment:
We are not externally mentioning any return right.

Reply:
Yeah, this does not make sense. It works perfect now.